src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,58 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        ArrayList<DataFromFile> dataList = new ArrayList<>();
+        getDataFile(fromFileName, dataList);
+        writeToFile(toFileName, calculating(dataList));
+    }
+
+    private void getDataFile(String fromFileName, ArrayList<DataFromFile> dataList) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line = null;
+            DataFromFile data = null;
+            while ((line = reader.readLine()) != null) {
+                String[] strings = line.split(",");
+                data = new DataFromFile(strings[0], Integer.valueOf(strings[1]));
+                dataList.add(data);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file", e);
```

Comment:
```suggestion
            throw new RuntimeException("Can't read data from file: " + fromFileName, e);
```
by adding info about file, you are making exceptions more informative